shm/cargo.cpp

```diff
@@ -0,0 +1,16 @@
+#include "cargo.hpp"
+
+Cargo();
+Cargo(size_t amount, std::string name, size_t base_price)
+    : amount_(amount), name_(name), base_price_(base_price)
+{}
+~Cargo() {;} 
+
+std::string getName() const { return name_; }
+size_t getAmount() const { return amount_; }
+size_t getBase_price() const { return base_price_; }
```

Comment:
Let's do not mix two naming conventions
```suggestion
size_t getBasePrice() const { return base_price_; }
```